hazelcast-documentation/src/SpringIntegration.md

```diff
@@ -228,6 +228,7 @@ Hazelcast Distributed `ExecutorService`, or more generally any Hazelcast managed
 
   <hz:hazelcast id="instance">
     <hz:config>
+      <hz:spring-aware />
```

Comment:
@bilalyasar , this is a spring convention. See http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/transaction/annotation/EnableTransactionManagement.html
You enable `Transactional` annotation by adding `<tx:annotation-driven />` for example.